cg/meta/workflow/analysis.py

```diff
@@ -423,3 +428,31 @@ def parse_analysis(
         """Parses output analysis files"""
 
         raise NotImplementedError
+
+    def clean_analyses(self, case_id: str) -> None:
+        """Add a cleaned at date for all analyses related to a case."""
+        analyses: list = self.status_db.family(case_id).analyses
+        LOG.info(f"Adding a cleaned at date for case {case_id}")
+        for analysis_obj in analyses:
+            analysis_obj.cleaned_at = analysis_obj.cleaned_at or dt.datetime.now()
+            self.status_db.commit()
+
+    def clean_run_dir(self, case_id: str, yes: bool, case_path: Union[List[Path], Path]) -> int:
+        """Remove workflow run directory."""
+
+        try:
+            self.verify_case_path_exists(case_id=case_id)
+        except FileNotFoundError:
+            self.clean_analyses(case_id)
+
+        if yes or click.confirm(f"Are you sure you want to remove all files in {case_path}?"):
+            if case_path.is_symlink():
+                LOG.warning(
+                    f"Will not automatically delete symlink: {case_path}, delete it manually",
+                )
+                return EXIT_FAIL
+
+            shutil.rmtree(case_path, ignore_errors=True)
```

Comment:
Should we be ignoring errors related to files that could not be removed? Any reason why this is set to True?  I know you did not change this behaviour but I wonder if this is what we want.

Reply:
I believe the reason is not to make it crash if it encounters errors. So if it can't delete a file due to permission reasons, for example, it will still preceded with the other files instead of crashing.

Reply:
Yes, that I know, but I was thinking that maybe we we would like to know when that happens (i.e. we would like to know if files cannot be deleted) since I guess it should not happen generally. But maybe this is out of the scope of this pr